layers/+checkers/syntax-checking/packages.vim

```diff
@@ -1,6 +1,2 @@
-if g:spacevim.vim8 || g:spacevim.nvim
-  MP 'w0rp/ale',  { 'on': [] }
-  call timer_start(200, 'spacevim#defer#ale')
-else
-  MP 'vim-syntastic/syntastic',     { 'on': 'SyntasticCheck' }
-endif
+MP 'dense-analysis/ale'
```

Comment:
Why remove `syntastic`?

Reply:
Sorry, the last commit was for my own config. I've revert it back.